modules/ossm-release-notes-3-1-technology-preview-features.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-1-technology-preview-features_{context}"]
+= {SMProductName} version 3.1 Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+
+[id="istio-ambient-mode-rn_{context}"]
+== Istio ambient mode
+
+This release updates the status of {istio} ambient mode to Technology Preview. Istio ambient mode provides a sidecar-less alternative data plane to the traditional sidecar-based data plane. With ambient mode, the data plane splits into node-level L4 ZTunnels and namespace (by default) scoped L7 Waypoint proxies.
+
+As {istio} ambient mode requires the use of {k8s} Gateway API custom resource definitions (CRDs), use {product-title} 4.19 or later, which includes these features by default.
+
+To avoid potential conflicts, you must install {istio} ambient mode only on clusters that do not have an existing {SMProductName} installation. {istio} ambient mode is not compatible with clusters that use {SMProductName} 2.6 or earlier.
+
+When using {istio} ambient mode, pods that rely on `liveness` or `readiness` probes require the OVN-Kubernetes gateway mode to be set to `local` instead of the default `shared` mode. In local mode, the traffic routes through the host and is processed using the routing table of the host, ensuring that probes function correctly. For more information, see the "Configuring gateway mode" section in the OVN-Kubernetes documentation.
```

Comment:
```suggestion
When you use {istio} ambient mode, pods that rely on `liveness` or `readiness` probes require you to set the OVN-Kubernetes gateway mode to `local` instead of the default `shared` mode. In local mode, traffic routes through the host and the host processes it using the routing table, ensuring that probes function correctly. For more information, see the "Configuring gateway mode" section in the OVN-Kubernetes documentation.
```